Asterix/tests/test_coronagraph.py

```diff
@@ -37,6 +32,19 @@ def test_all_coronagraphs():
             assert np.allclose(coro_psf, np.transpose(coro_psf), atol=atols[i],
                                rtol=0), f"Coronagraphic image is not symmetric in transpose for '{coro}'."
 
+    # add test polychromatic. We do not add any performance tests because most coronagraphs aren't great 
+    # in polychromatic, but it should at least run. 
+    modelconfig.update({"Delta_wav": 30e-9})
+    modelconfig.update({"nb_wav": 3})
+    for i, coro in enumerate(coros_to_test):
+        
+        # Create the coronagraph
+        corono = Coronagraph(modelconfig, Coronaconfig)
+        try:
+            coro_psf = corono.todetector_intensity(center_on_pixel=True, in_contrast=True)
+        except:
+            assert False, f"Coronagraph '{coro}' does not work in polychromatic light."
```

Comment:
Why did you opt for a try-except here? If the propagation fails, it will throw an error and the test fails anyway.

Reply:
ok, I thought we needed absolutely to have a assert in the test